package.json

```diff
@@ -12,34 +12,34 @@
     "build:eleventy": "eleventy --quiet",
     "lint": "stylelint \"./src/**/*.scss\" --cache --cache-location .cache/.stylelintcache",
     "dev": "cross-env NODE_ENV=development ELEVENTY_ENV=development npm-run-all clean build:styles --parallel start:* serve",
-    "start:styles": "sass docs/assets -wo dist/css/",
+    "start:styles": "sass docs/assets -w dist/css/",
```

Comment:
@lnolte what did -o do in node-sass?
We might want to add the dart-sass equivalent.

Reply:
`-o` set the output path which seems to be implicit in dart-sass.

From reading up on their docs `sass --watch docs/assets:dist/css` should be the correct command here